config/environments/test.rb

```diff
@@ -39,4 +39,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Clearance
+  config.action_mailer.default_url_options = { :host => "test.locahost://3000" }
```

Comment:
Prefer Ruby > 1.9 hash syntax? `host: "test.localhost://3000"`

Why the `//` before `3000`?

(This comment applies to the other config files as well)